proto/collection.proto

```diff
@@ -162,4 +173,5 @@ message Collection {
     optional Palette palette_dark = 3;
     optional Branding branding = 6;
     optional bool premium_content = 7;
+    FollowUp follow_up = 8;
```

Comment:
I suppose we'll always have follow up links for every collection?

Reply:
Previously I thought that we were always able to get the list URL for a collection, but on second thought we may have _container_ on a front which is not a collection set in the front tools (like the crossword container).  It may be more flexible to make it optional.

Reply:
Yeah, that sounds good! It leaves us open to handling different types of containers in the future too (that may intentionally/unintentionally not have a follow_up object) :)